hbase-client/src/main/java/org/apache/hadoop/hbase/client/MetricsConnection.java

```diff
@@ -598,6 +611,24 @@ public void incrementServerOverloadedBackoffTime(long time, TimeUnit timeUnit) {
     overloadedBackoffTimer.update(time, timeUnit);
   }
 
+  /** incr */
```

Comment:
is there a chance for a test which tests these metrics? Something like what exists in `TestMetaCache#testUserRegionLockThrowsException`?

Reply:
+1, unit test required

Reply:
Added some assert on counts of those metrics, it's difficult to assert on the times. Let me know if you have any opinion. @d-c-manning @apurtell